config/notifiers.go

```diff
@@ -127,6 +127,16 @@ var (
 		Expire:   duration(1 * time.Hour),
 		HTML:     false,
 	}
+
+	// DefaultSNSConfig defines default values for SNS configurations.
+	DefaultSNSConfig = SNSConfig{
+		NotifierConfig: NotifierConfig{
+			VSendResolved: true,
+		},
+		APIVersion:  "sns.default.api_version",
```

Comment:
What's `sns.default.api_version`? Did you want it to be a template?

Reply:
I'm not sure about changing the api version; the API Version is hard coded [in the SNS SDK client metadata currently.](https://github.com/aws/aws-sdk-go/blob/v1.38.35/service/sns/service.go#L66) Is this something we would want to change?

Reply:
> I'm not sure about changing the api version

What's part of the specifications. If you suggest to remove it, I would raise the topic in the issue.

Getting back to my comment: it was about the default value being `sns.default.api_version`. That's just a static string, but who does interpret it? I'm not sure I understand it.

Reply:
>  If you suggest to remove it, I would raise the topic in the issue.
I think since it's hard coded in the API, it makes the most sense to remove it. I'll make that change and add it to the issue.